app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
What is this comment form for? You have the form for adding a new comment added on your posts show page